UI_Engine/UI_Engine.csproj

```diff
@@ -64,6 +64,10 @@
       <HintPath>C:\ProgramData\BHoM\Assemblies\Reflection_oM.dll</HintPath>
       <Private>False</Private>
     </Reference>
+    <Reference Include="Serialiser_Engine, Version=3.0.0.0, Culture=neutral, processorArchitecture=MSIL">
+      <SpecificVersion>False</SpecificVersion>
+      <HintPath>..\..\..\..\..\..\..\ProgramData\BHoM\Assemblies\Serialiser_Engine.dll</HintPath>
```

Comment:
Reference should be set to `ProgramData` - probably explains why CheckCore is failing.

Reply:
Now fixed